tests/testthat/_snaps/zenodo.md

```diff
@@ -0,0 +1,54 @@
+# download_zenodo() works for a single-file record
+
+    Code
+      download_zenodo(doi = "10.5281/zenodo.3784149", path = zenodo_dir)
+    Message
```

Comment:
When I run the tests locally, the lines `    Message` are replaced by `    Message <simpleMessage>`.

Reply:
Interesting. Does {testthat} consider this to be a change, and request to accept such change? In that case, seems like an issue with snapshotting.

Reply:
Yes indeed. It suggested to rerun the test in some kind of review mode. If I did that, I could either skip or accept the changes to the md file.

Reply:
Can you run this and paste the output? That would be useful for an issue at {testthat}.

```r
reprex::reprex({
  library(testthat)
  test_that("snapshotting works", {
    local_edition(3)
    expect_snapshot(message("hi"))
  })
}, session_info = TRUE)
```

Reply:
I've extended the example.

Reply:
Thanks. For comparison:

``` r
library(testthat)
test_that("snapshotting runs", {
  local_edition(3)
  expect_snapshot(error = TRUE, {
    1 + 1
    message("hi")
    warning("a warning")
    "a" + 1
  })
})
#> Can't compare snapshot to reference when testing interactively.
#> ℹ Run `devtools::test()` or `testthat::test_file()` to see changes.
#> Current value:
#> Code
#>   1 + 1
#> Output
#>   [1] 2
#> Code
#>   message("hi")
#> Message
#>   hi
#> Code
#>   warning("a warning")
#> Condition
#>   Warning:
#>   a warning
#> Code
#>   "a" + 1
#> Condition
#>   Error in `"a" + 1`:
#>   ! non-numeric argument to binary operator
#> ── Skip: snapshotting runs ─────────────────────────────────────────────────────
#> Reason: empty test
```

<sup>Created on 2023-11-21 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

<details style="margin-bottom:10px;">
<summary>
Session info
</summary>

``` r
sessioninfo::session_info()
#> ─ Session info ───────────────────────────────────────────────────────────────
#>  setting  value
#>  version  R version 4.3.2 (2023-10-31)
#>  os       Linux Mint 21.2
#>  system   x86_64, linux-gnu
#>  ui       X11
#>  language nl_BE:nl
#>  collate  nl_BE.UTF-8
#>  ctype    nl_BE.UTF-8
#>  tz       Europe/Brussels
#>  date     2023-11-21
#>  pandoc   3.1.1 @ /usr/lib/rstudio/resources/app/bin/quarto/bin/tools/ (via rmarkdown)
#> 
#> ─ Packages ───────────────────────────────────────────────────────────────────
#>  package     * version date (UTC) lib source
#>  brio          1.1.3   2021-11-30 [3] RSPM (R 4.2.0)
#>  cli           3.6.1   2023-03-23 [3] RSPM (R 4.2.0)
#>  desc          1.4.2   2022-09-08 [3] RSPM (R 4.2.0)
#>  digest        0.6.33  2023-07-07 [3] RSPM (R 4.2.0)
#>  evaluate      0.23    2023-11-01 [3] RSPM (R 4.3.0)
#>  fansi         1.0.5   2023-10-08 [3] RSPM (R 4.3.0)
#>  fastmap       1.1.1   2023-02-24 [3] RSPM (R 4.2.0)
#>  fs            1.6.3   2023-07-20 [3] RSPM (R 4.2.0)
#>  glue          1.6.2   2022-02-24 [3] RSPM (R 4.2.0)
#>  htmltools     0.5.7   2023-11-03 [3] RSPM (R 4.3.0)
#>  knitr         1.45    2023-10-30 [3] RSPM (R 4.3.0)
#>  lifecycle     1.0.4   2023-11-07 [3] RSPM (R 4.3.0)
#>  magrittr      2.0.3   2022-03-30 [3] RSPM (R 4.2.0)
#>  pillar        1.9.0   2023-03-22 [3] RSPM (R 4.2.0)
#>  pkgload       1.3.3   2023-09-22 [3] RSPM (R 4.3.0)
#>  purrr         1.0.2   2023-08-10 [3] RSPM (R 4.2.0)
#>  R.cache       0.16.0  2022-07-21 [3] RSPM (R 4.2.0)
#>  R.methodsS3   1.8.2   2022-06-13 [3] RSPM (R 4.2.0)
#>  R.oo          1.25.0  2022-06-12 [3] RSPM (R 4.2.0)
#>  R.utils       2.12.2  2022-11-11 [3] RSPM (R 4.2.0)
#>  R6            2.5.1   2021-08-19 [3] RSPM (R 4.2.0)
#>  reprex        2.0.2   2022-08-17 [3] RSPM (R 4.2.0)
#>  rlang         1.1.2   2023-11-04 [3] RSPM (R 4.3.0)
#>  rmarkdown     2.25    2023-09-18 [3] RSPM (R 4.3.0)
#>  rprojroot     2.0.4   2023-11-05 [3] RSPM (R 4.3.0)
#>  rstudioapi    0.15.0  2023-07-07 [3] RSPM (R 4.2.0)
#>  sessioninfo   1.2.2   2021-12-06 [3] RSPM (R 4.2.0)
#>  styler        1.10.2  2023-08-29 [3] RSPM (R 4.2.0)
#>  testthat    * 3.2.0   2023-10-06 [3] RSPM (R 4.3.0)
#>  utf8          1.2.4   2023-10-22 [3] RSPM (R 4.3.0)
#>  vctrs         0.6.4   2023-10-12 [3] RSPM (R 4.3.0)
#>  waldo         0.5.2   2023-11-02 [3] RSPM (R 4.3.0)
#>  withr         2.5.2   2023-10-30 [3] RSPM (R 4.3.0)
#>  xfun          0.41    2023-11-01 [3] RSPM (R 4.3.0)
#>  yaml          2.3.7   2023-01-23 [3] RSPM (R 4.2.0)
#> 
#>  [1] /home/floris/lib/R/library
#>  [2] /usr/local/lib/R/site-library
#>  [3] /usr/lib/R/site-library
#>  [4] /usr/lib/R/library
#> 
#> ──────────────────────────────────────────────────────────────────────────────
```

</details>

Reply:
OK, so it's b/c I had an older testthat. After updating I see the same as your output.

Reply:
@hansvancalster can you update {testthat} to 3.2.0 (current) and re-run? Wouldn't be surprised this is another effect of  versions :slightly_smiling_face: .

Reply:
I guess you haven't seen [my comment](https://github.com/inbo/inborutils/pull/124#discussion_r1400574832) 😉 ?

Reply:
:slightly_smiling_face: so far the automatically updating GitHub webpages. Your confirmation appeared later here. Thanks and solved! I'll stay with my hypothesis is that `expect_snapshot()` is useful for evaluating more extended function output.